test/examples/test_examples.py

```diff
@@ -25,7 +24,43 @@ def test_basic_configuration():
                 received += 1
                 continue
 
-        if sent * 2 == received:
+        if sent != 0 and received != 0 and sent * 2 == received:
+            ret = True
+        else:
+            raise subprocess.CalledProcessError(1, '')
+
+    except subprocess.CalledProcessError:
+        for l in out:
+            print(l)
+    except subprocess.TimeoutExpired:
+        print('TIMEOUT')
```

Comment:
I just realized that in case of timeout we are not printing and we probably should. Same goes for the hello_world example test

Reply:
I think that timeout exception makes out NOT to be populated.
We can try to print `out` but I would guess that nothing would be printed.